buildenv/jenkins/variables/defaults.yml

```diff
@@ -600,59 +356,39 @@ x86-32_windows:
   excluded_tests:
     8:
       - special.system
-    11:
```

Comment:
This shouldn't be adding Windows jdk11 special.system testing.

Reply:
This is win32 so it doesn't apply which is why I am removing the exclude.